### PR TITLE
Engine fire: add listeners, hydraulic simulation

### PR DESCRIPTION
**Pull request by legoboyvdlp**
as requested

### CODE REVIEW

**Comment by legoboyvdlp** [comment]
Hydraulics coming soon!

File: Nasal/systems.nas [hunk 1]
```diff
@@ -31,4 +31,9 @@ setlistener("/b737/controls/gear/lever", func {
 		setprop("/controls/gear/gear-down", 1);
 		setprop("/b737/controls/gear/lever", 0);
 	} 
+});
+
+
+setlistener("/sim/signals/fdm-initialized", func {
+	boeing737.hyd_init();
```
Comment (zakalawe) [inline Nasal/systems.nas:38]: I think you need to remove this as well, or we'll get a Nasal error.

Comment (zakalawe) [reply Nasal/systems.nas:38]: Or just leave it in until the hydraulics are done.

**Comment by legoboyvdlp** [comment]
There's the hydraulics and FCS. As far as I see everything works (including HYD warning if the pumps are on and there is no pressure).

**Comment by zakalawe** [comment]
Hey this looks okay, just I need to be careful to update my input bindings, since I I have hardware to set spoilers. I also have a local fix to make split throttles work, there's a missing rule in the auto-throttle rule XML. Again I just need to push the fix so it doesn't get lost if you're working in the same area.

**Comment by legoboyvdlp** [comment]
Awesome, I'll pull from your repository then :)

**Pull request merged**